map-poller.php

```diff
@@ -61,7 +61,9 @@
     {
         while (($file = readdir($dh)) !== false)
         {
-            if ("." != $file && ".." != $file && ".htaccess" != $file && "index.php" != $file)
+            $allowed = array('conf');
+            $ext = pathinfo($file, PATHINFO_EXTENSION);
+            if (!in_array($ext, $allowed))
```

Comment:
I think the `!` is wrong here as we want to get all file which *have* the `.conf` extension. don't we?

Reply:
I guess
```
if (pathinfo($file, PATHINFO_EXTENSION) == "conf")
```
would do the trick?